src/main/java/edu/cornell/kfs/module/cam/rest/resource/CuCapAssetInventoryApiResource.java

```diff
@@ -184,18 +187,44 @@ public Response updateAsset(@PathParam(CuCamsConstants.CapAssetApi.ASSET_TAG_PAR
             return ex instanceof BadRequestException ? respondBadRequest() : respondInternalServerError(ex);
         }
     }
+    
+    @GET
+    @Path("/campusCodes")
+    public Response getCampusCodes(@Context HttpHeaders headers) {
+        try {
+            List<Map<String, String>> campusCodes = new ArrayList<>();
+            
+            Map<String, Object> fieldValues = new HashMap<>();
+            fieldValues.put(CuCamsConstants.CapAssetApi.ACTIVE, true);
+            Collection<org.kuali.kfs.sys.businessobject.Campus> campuses = 
+                SpringContext.getBean(BusinessObjectService.class).findMatching(
+                    org.kuali.kfs.sys.businessobject.Campus.class, fieldValues);
```

Comment:
Is there a particular reason why the fully qualified `Campus` classname is being used throughout this file? Can an import be used instead, or is there already a conflicting classname in use?